examples/dmd/local_dw_csv.cpp

```diff
@@ -0,0 +1,889 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Compile with: make local_dw_csv
+//
+// Generate CSV databse on heat conduction with: heat_conduction_csv.sh
+//
+// =================================================================================
+//
+// Local serial DMD command:
+//   mpirun -np 8 local_dw_csv -o local_csv_serial -rdim 16 -dtc 0.01
+//
+// Final-time prediction error (Last line in run/local_csv_serial/hc_par5_prediction_error.csv):
+//   0.0004063242226265
+//
+// Local time windowing DMD command:
```

Comment:
distance windowing

Reply:
The examples in the new scripts are now removed.